windows-release.ps1

```
@@ -168,6 +214,19 @@ if ($CMAKE_BUILD_TYPE -eq "Release") {
     Copy-Item "$ICU_STATIC_LIBRARY/icuuc.lib" "$output/lib/sicuucd.lib" -Force
 }
 
+# Copy ASAN runtime DLLs to output when sanitizers are enabled
+if ($ENABLE_SANITIZERS -and $ClangRtLibPath) {
+    $asanArch = if ($Platform -eq "ARM64") { "aarch64" } else { "x86_64" }
+    $asanDll = "clang_rt.asan_dynamic-${asanArch}.dll"
+    $asanDllPath = Join-Path $ClangRtLibPath $asanDll
+    if (Test-Path $asanDllPath) {
+        Copy-Item $asanDllPath "$output/bin/$asanDll" -Force
+        Write-Host ":: Copied ASAN runtime DLL: $asanDll"
```

PSScriptAnalyzer (1.24.0)

[warning] File 'windows-release.ps1' uses Write-Host. Avoid using Write-Host because it might not work in all hosts, does not work when there is no host, and (prior to PS 5.0) cannot be suppressed, captured, or redirected. Instead, use Write-Output, Write-Verbose, or Write-Information.

(PSAvoidUsingWriteHost)